File: frontend/src/app/api/models/flight-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from './time-place-rm';
export interface FlightRm {
  airline?: null | string;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  id?: string;
  price?: number;
  remainingSeats?: number;
}

File: backend/models/FlightRm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend.models
{
    public record FlightRm(
        Guid Id,
        string Airline,
        double Price,
        TimePlaceRm Departure,
        TimePlaceRm Arrival,
        int RemainingSeats
    );
}